packages/fxa-settings/src/models/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './Session';
 export * from './hooks';
 export * from './verification';
 export * from './reliers';
+export * from './Legal';
```

Comment:
Unrelated @pdehaan -ism, but we should probably add a copyright header to this file.